classes/template.php

```diff
@@ -75,7 +75,7 @@ public function save($data) {
         $DB->update_record('customcert_templates', $savedata);
 
         // Only trigger event if the name has changed.
-        if ($savedata->name != $data->name) {
```

Comment:
So this was never being triggered since they would be the same. I think this needs a unit test to verify it is getting triggered then we would have caught this. Also, please use the method `get_name()`. *sigh* Looking at this plugin's code I wish I could go back in time and rewrite it but I guess thats software development for you!

Reply:
Added PHPUnit tests test_updating_a_template and test_updating_a_template_no_change.